dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/converter/NotesConverterService.java

```diff
@@ -68,8 +69,9 @@ public TrackedEntityComment from( TrackerPreheat preheat, Note note )
         comment.setAutoFields();
         comment.setCommentText( note.getValue() );
 
-        // FIXME: what about the storedBy and lastUpdatedBy -> currently they are set to
-        // null
+        comment.setLastUpdatedBy( preheat.getUser() );
+        comment.setLastUpdated( new Date() );
```

Comment:
This is more like a general issue for all these lastUpdated, created etc. fields: shall we instead use default values at the DB level? What do you think?

Reply:
yes probably it would be much better to have them out of the box, either by default value in DB or default values during instance creation.